test/conformance/helpers/broker_tracing_test_helper.go

```diff
@@ -153,23 +153,20 @@ func setupBrokerTracing(
 	// 9. Broker Filter for the "transformer" trigger sends the event to the transformer pod.
 	// 10. Transformer pod receives the event from the Broker Filter for the "transformer" trigger.
 	// 11. Broker Filter for the "transformer" sends the transformer pod's reply to the Broker
-	//     InChannel (Ingress Channel).
-	// 12. Broker InChannel receives the event from the Broker Filter for the "transformer" trigger.
-	// 13. Broker InChannel sends the event to the Broker Ingress.
-	// 14. Broker Ingress receives the event from the Broker InChannel.
-	// 15. Broker Ingress sends the event to the Broker's TrChannel.
-	// 16. Broker TrChannel receives the event from the Broker Ingress.
-	// 17. Broker TrChannel sends the event to the Broker Filter for the "transformer" trigger.
-	//     18. Broker Filter for the "transformer" trigger receives the event from the Broker
+	//     Ingress.
+	// 12. Broker Ingress receives the event from the Broker Filter for the "transformer" trigger.
+	// 13. Broker Ingress sends the event to the Broker's TrChannel.
+	// 14. Broker TrChannel receives the event from the Broker Ingress.
+	// 15. Broker TrChannel sends the event to the Broker Filter for the "transformer" trigger.
+	//     16. Broker Filter for the "transformer" trigger receives the event from the Broker
```

Comment:
nit: indentation

Reply:
I think this was intentional as I was just changing the index numbers. But I'm not really sure why the additional indent.